package.json

```diff
@@ -51,6 +51,9 @@
         ],
         "**/lwc/**": [
             "eslint"
+        ],
+        "**/aura/**": [
```

Comment:
Shouldn't it be `**/{aura|lwc}/**`?

Reply:
Yeah I guess we can use that. It's a bit less readable but it might be more efficient since we would only launch eslint once.